preprocessors/collage-detector/detect.py

```diff
@@ -91,7 +91,8 @@ def detect_collage():
     except jsonschema.exceptions.ValidationError as e:
         logging.error(e)
         return jsonify("Invalid Preprocessor JSON format"), 500
-    logging.debug("Sending response")
+    logging.debug("Collage Detector Response")
+    logging.debug(response)
```

Comment:
If this will output on two separate lines, should we combine into one to allow for easier grep, etc.?

Reply:
Addressed this a bit in an earlier comment, but using a named logger could help. Docker compose does give some naming in the overall compose logs. Including some below:
```
content-categoriser-1          | [2024-06-19 11:39:05 +0000] [9] [DEBUG] POST /preprocessor
orchestrator-1                 | Sending to preprocessor "object-sorting"
```

Reply:
(To be clear, I think this is likely sufficient for our purposes?)